discovery_engine_core/web-api/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM debian:bookworm-slim
+
+WORKDIR /service
+
+COPY ./ ./
+
+RUN chmod +x ./web-api
+VOLUME /service/assets/data.json
```

Comment:
@jorge-cr-13 is it ok for you if we provide the json file with data this way? so we can just restart the container with updated data instead of bundling it with the image

Reply:
This `data.json` for doing a backup of the state of the container?

Reply:
Actually, I don't understand what is the plan of this `data.json` file. Should we store it somewhere so everytime the container has been restored it needs to find this file and restore it from here?

Reply:
the `data.json` file is source of internal state for the web service. At the startup we read the contents of this file, and calculate some additional state from that. So this file is read only. It's needed so that the api can return something. The reason it was made a volume it's to have a way of updating content of this file without the need to rebuild the image.

Reply:
Ok.. then yeah, I don't see a problem with using the file as a `data.json`.